tests/testthat/test-derive_params_growth_age.R

```diff
@@ -1,41 +1,398 @@
 # derive_params_growth_age ----
+library(dplyr)
```

Comment:
```suggestion
```

Per [textbook](https://r-pkgs.org/testing-basics.html#test-organisation)  you'll want to avoid calling libraries in unit-test suite, if you need to use {pkgname} functions just use `pkgname::xxx()` format, demonstrated below with the `tibble::tribble()` call. Although it's ideal to this with all outside-functions, I think `dplyr` ones are so commonly used we can use the `importFrom` in the `admiralpeds-package.R` to avoid excessive use